app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitViewFragment.java

```diff
@@ -20,6 +28,7 @@
 public class HabitViewFragment extends Fragment {
 
     private Habit _habit;
+    private final User _user = new User("TestUser", "123");
 
```

Comment:
We should pass in the user from the HabitListFragment rather than creating a new one here.

Reply:
I agree ^

Reply:
- [ ] pass in user from `HabitListFragment`